caliper_sender.py

```diff
@@ -75,11 +75,11 @@ def send_caliper_event():
     cron_job = 'test_cron'
     last_runtime = get_last_runtime(cron_job)
     events = fetch_events(last_runtime)
-
-    print (events)
+    batch = []
+    batch_size = 5
```

Comment:
Batch size should be configured via environment variable
```suggestion
    batch_size = os.getenv("BATCH_SIZE", 5)
```

Also add this to the .env.sample